src/Ganeti/HTools/ExtLoader.hs

```diff
@@ -122,8 +122,12 @@ loadExternalData opts = do
   now <- getClockTime
 
   let ignoreDynU = optIgnoreDynu opts
+      nodeMem = optStaticKvmNodeMemory opts
       eff_u = if ignoreDynU then [] else util_data
       ldresult = input_data >>= (if ignoreDynU then clearDynU else return)
+                            >>= (if nodeMem >= 0
```

Comment:
Sure, it will be useful for merging. However now I am not entirely sure if this will be temporary only. The feature is handled in 2.17 for the Luxi backend only while this flag works universally for all backends. I think it should be not removed before the other backends can handle this correctly as well.